components/load_from_files/fondant_component.yaml

```diff
@@ -0,0 +1,21 @@
+name: Load from files
+description: Component that loads a dataset from files
+image: ghcr.io/ml6team/load_from_files:dev
+
+produces:
+  file:
+    fields:
+      content:
+        description: content of the file in binary form
+        type: binary
+      id:
```

Comment:
We don't need to include the `id` field here, the index is handled separately.

Reply:
ok Cool, Ill remove that